cgo/lapack.go

```diff
@@ -13,6 +13,7 @@ import (
 
 // Copied from lapack/native. Keep in sync.
```

Comment:
Can we reflect this comment in the native location as well - I should have done that.